.gitignore

```diff
@@ -1,4 +1,17 @@
 _site
 config/webadmit_path.txt
+*.bz2
+*.log
+*.rdb
+*.swp
+*~
+.bundle
 .DS_Store
+.env
+.env.local
+.idea/
+.powenv
+.powrc
+.sass-cache
+.yardoc/
```

Comment:
At least some of these should be in your global gitignore, not in this repo's gitignore.

Are any of these necessary for this repo?